build-aux/com.obsproject.Studio.json

```diff
@@ -6,7 +6,7 @@
     "command": "obs",
     "finish-args": [
         "--socket=wayland",
-        "--socket=x11",
```

Comment:
Keep unresolved until the commit tittle is changed.

Reply:
I've fixed the commit message and PR title.